code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Medicine.dm

```diff
@@ -691,7 +701,16 @@
 			if(prob(33))
 				H.Confuse(10)
 
-/datum/reagent/nanoperidaxon
+/datum/reagent/peridaxon/overdose(var/mob/living/carbon/M, var/alien, var/removed) //Oversoing on Peridaxon greatly reduces its organ healing effects
+	if(ishuman(M))
+		var/mob/living/carbon/human/H = M
+		for(var/obj/item/organ/I in H.internal_organs)
+			if(I.robotic >= ORGAN_ROBOT)
+				continue
+			if(I.damage > 0)
+				I.damage = max(I.damage + 0.75*removed, 0) //When overdosed Peri heals at 1/4th its normal rate
+
+/datum/reagent/nanoperidaxon //I really don't know how I feel about this chem...
```

Comment:
why do you need to tell all future coders that you don't know how to feel about this chem.